### DISCUSSION TITLE
How to write the receive data to file?

### DISCUSSION THREAD

**Discussion by panda-mo** [dannagle/PacketSender#343] [Q&A]
PacketSender is a great tool !

Here is my problem. I use the software on PC to send binary files to a target machine, and the machine return data to PC. Just wondering how to write the received data to a file? 

Thanks in advance for your assistance.

**Comment by dannagle** [comment]
I am happy you like Packet Sender! 😄 

Auto-saving logs has been requested a few times before:  #163 

There are several ideas requested logging. Auto-saving is the big one though, and it is not that complicated to add. I was trying to add an improved Smart Responses, but I may attempt logs instead for the next release. It is easier and it would really help test setups.

**Reply by panda-mo** [reply]
OK, I see. Thanks for your quick reply.
